src/exporters/__init__.py

```diff
@@ -1 +1,3 @@
 from .cds import CDSExporter
+
```

Comment:
what do these do in the `__init__.py` files?

Reply:
They make imports easier. This way, when we are running code from this library, we can do
`from src.exporters import CDSExporter` 
instead of
`from src.exporters.cds import CDSExporter`

It's especially useful when we have multiple exporters:

`from src.exporters import CDSExporter, otherExporter, ....` 
instead of
`from src.exporters.cds import CDSExporter`
`from src.exporters.other import OtherExporter`
`...`

Reply:
We should probably think of a good way to differentiate between different exporters.
Explicitly calling the class doesn't seem like the best approach; ideally there would be a way to automatically work out which data store (and therefore which exporter) is being used

Reply:
so a different exporter for what each ERA5 data-store?
- [Monthly Averaged Single Level Data](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means?tab=overview)
- [Monthly Averaged Pressure Levels](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-pressure-levels-monthly-means?tab=overview)
- [Hourly Single Levels](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels)
- [Monthly Single Levels](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-pressure-levels)

Or do you mean for something else?

Reply:
We'd have one for the ERA5 data store, since it has a single API, must another one for Earth Engine (for example)

Reply:
Yes that makes sense. EE is great because hte data all COMES in teh same format but it does tie you to having google cloud storage OR google compute resources. 

An alternative is downloading scripts from the original data providers (e.g. NASA for VHI, MODIS etc.) but that could be even more ugly having to preprocess these separately.

@gabrieltseng  What do you think?